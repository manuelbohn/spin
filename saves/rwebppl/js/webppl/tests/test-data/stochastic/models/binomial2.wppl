var model = function() {
  return binomial(0.4, 2);
};
