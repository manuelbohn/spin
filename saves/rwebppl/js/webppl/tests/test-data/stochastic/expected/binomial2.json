{
  "hist": {
    "0": 0.36,
    "1": 0.48,
    "2": 0.16
  }
}
